course/page/code.py

```diff
@@ -443,6 +443,12 @@ class CodeQuestion(PageBaseWithTitle, PageBaseWithValue):
         based on its :attr:`access_rules` (not the ones of the flow), a warning
         is shown. Setting this attribute to True will silence the warning.
 
+    .. attribute:: docker_image
+
+        Optional.
+        Specific Docker image within which to run code for the participants
+        answer.
```

Comment:
* Typo: participant's
* This should make brief reference to the interface that the Docker image is expected to offer. (to counter a notion that any docker image will do)
* Also state whether the image needs to be already downloaded or whether it will be downloaded on demand.

Reply:
>Specific Docker image within which to run code for the participants
        answer.  This overrides the image set in the `local_settings.py`
        configuration.  The Docker image should provide two files; these are
        supplied in RELATE's standard Python Docker image by `course/page/
        code_run_backend_python.py` and `course/page/code_feedback.py`, for
        instance.  Consult `docker-image-run-py/docker-build.sh` for one
        example of a local build.  The Docker image should already be loaded
        on the system (RELATE does not pull the image automatically).

I've made these changes and added the above explanation.